README.md

```diff
@@ -174,19 +174,6 @@ not a curated list of Julia packages. In particular this means that:
    you install through the General registry -- you are responsible for reviewing your
    code dependencies.
 
-## Tips for registry maintainers
```

Comment:
Maybe add a pointer to RegistryCI docs?

Reply:
Hm, they are linked earlier in the automerge section, though that's kind of a different context. I'm not totally sure General is the right place though-- maybe the [Pkg.jl docs on registries](https://pkgdocs.julialang.org/v1/registries/) should be updated to mention RegistryCI (and probably LocalRegistry.jl too)

Reply:
I made a PR to do that here: https://github.com/JuliaLang/Pkg.jl/pull/2659